src/nrepl/index.ts

```diff
@@ -447,7 +447,13 @@ export class NReplSession {
         return new Promise<cider.TestResults>((resolve, reject) => {
             let id = this.client.nextId;
             this.messageHandlers[id] = resultHandler(resolve, reject);
-            this.client.write({ op: "test-all", id, session: this.sessionId, "load?": true });
+            this.client.write({
```

Comment:
Hrm, actually this might be a behaviour change - since we are no longer passing `load?` true. Let me confirm...

Reply:
There doesn't seem to be a change in behavior, as described further in the main PR thread.